src/components/StoryBook/StoriesByYear.tsx

```diff
@@ -36,3 +41,23 @@ const Container = styled.div`
 const Year = styled.div`
   width: 4rem;
 `;
+
+const CardsContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  overflow-x: auto;
+  padding: 1rem;
+
+  > div {
+    flex: 0 0 auto;
+  }
+`;
```

Comment:
이거 적용 되나요?

```suggestion
 & > div {
    flex: 0 0 auto;
  }
`;
```

이렇게해야하지않나요? 궁금해서.. 이렇게는 한번도 안 써봤었어서